src/packer.pkr.hcl

```
@@ -1,3 +1,20 @@
+packer {
+  required_plugins {
+    amazon = {
+      source  = "github.com/hashicorp/amazon"
+      version = "~> 1"
+    }
+    ansible = {
+      source  = "github.com/hashicorp/ansible"
+      version = "~> 1"
+    }
+  }
+  # The required_plugins section is only supported in Packer 1.7.0 and
+  # later.  We also want to avoid jumping to Packer v2 until we are
+  # ready.
+  required_version = ">=1.7.0,<2"
```

This should work based on [the documentation](https://developer.hashicorp.com/packer/docs/templates/hcl_templates/blocks/packer#-3).

```suggestion
  required_version = "~> 1.7"
```